core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionEvaluator.scala

```diff
@@ -29,15 +29,18 @@ object ExpressionEvaluator {
   }
 }
 
-class ExpressionEvaluator(slickEvaluator: SlickExpressionEvaluator, val rootEntities: Seq[EntityRecord]) {
+class ExpressionEvaluator(slickEvaluator: SlickExpressionEvaluator, val rootEntities: Option[Seq[EntityRecord]]) {
   def evalFinalAttribute(workspaceContext: SlickWorkspaceContext, expression: String): ReadWriteAction[Map[String, Try[Iterable[AttributeValue]]]] = {
     import slickEvaluator.parser.driver.api._
 
     JsonExpressionEvaluator.evaluate(expression) match {
       //if the expression evals as JSON, it evaluates to the same thing for every entity, so build that map here
-      case Success(parsed) => DBIO.successful((rootEntities map { entityRec: EntityRecord =>
-        entityRec.name -> Success(parsed)
-      }).toMap)
+      case Success(parsed) => DBIO.successful(rootEntities match {
+        case Some(entities) => (entities map { entityRec: EntityRecord =>
+          entityRec.name -> Success(parsed)
+        }).toMap
+        case None => Map("" -> Success(parsed))
```

Comment:
Out of curiosity - Why can we not pass in an empty Map here? (maybe explain the reasoning in an inline comment?)

Reply:
Also, it's interesting that Cromwell is fine with entity names being blank

Reply:
1) we can't pass in an empty map because we still need the evaluated values to send to cromwell.
2) we don't pass entity names to cromwell